lib/stack_master/http_parameter_resolver/strategy/one_per_line.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module StackMaster
+  module HttpParameterResolver
+    module Strategy
+      class OnePerLine
+        def accept
+          'text/plain'
+        end
+
+        def parse(response)
+          response.body.split(/[\r\n]+/)
+        end
```

Comment:
The `OnePerLine` response parsing strategy expects plain text and obtains one parameter per line in the response body.

Reply:
It should be easy to add more of these classes.